include/NovelRT/Ink/InkService.h

```diff
@@ -0,0 +1,36 @@
+// Copyright © Matt Jones and Contributors. Licensed under the MIT Licence (MIT). See LICENCE.md in the repository root for more information.
+
+#ifndef NOVELRT_INK_INKSERVICE_H
+#define NOVELRT_INK_INKSERVICE_H
+
+#ifndef NOVELRT_H
+#error Please do not include this directly. Use the centralised header (NovelRT.h) instead!
+#endif
+
+namespace NovelRT::Ink {
+  class InkService : public std::enable_shared_from_this<InkService> {
+  private:
+    struct Exports {
+      void(*Initialise)();
+      void(*Teardown)();
+      void(*GetStoryExports)(struct Story::Exports* exports);
+    };
```

Comment:
Minor nit, same as previous: Could we not use std::add_pointer<T>::type here to make these fnptrs easier to read?